test/jdk/javax/swing/regtesthelpers/JRobot.java

```diff
@@ -33,13 +33,13 @@
  */
 import java.awt.AWTException;
 import java.awt.Component;
-import java.awt.Dimension;
 import java.awt.EventQueue;
 import java.awt.Point;
 import java.awt.Rectangle;
 import java.awt.event.InputEvent;
-import java.awt.event.KeyEvent;
 import javax.swing.SwingUtilities;
+import java.util.concurrent.atomic.AtomicReference;
+import java.lang.reflect.InvocationTargetException;
```

Comment:
Could you sort the imports, please? You IDE should be able to handle it for you.